examples/simple_dashboard/bot.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import os
+from typing import Any
+
+import rillrate
+from rillrate import prime as rr_prime
+
+import hikari
+
+PREFIX = ","
+
+# Name used to group dashboards.
+# You could have multiple packages for different applications, such as a package for the bot
+# dashboards, and another package for a web server running alongside the bot.
+PACKAGE = "Package"
+# Dashboards are a part inside of package, they can be used to group different types of
+# dashboards that you may want to use, like a dashboard for system status, another dashboard
+# for cache status, and another one to configure features or trigger actions on the bot.
+DASHBOARD = "Dashboard"
+# This are menus inside the dashboard, you can use them to group specific sets
+# of data inside the same dashboard.
+GROUP_CONFIG = "1 - Config"
+# All of the 3 configurable namescapes are sorted alphabetically.
+
+
+def is_command(cmd_name: str, content: str) -> bool:
+    return content.startswith(f"{PREFIX}{cmd_name}")
+
+
+# This is how I do global data on the library, as Bot is generally accessible everywhere, so
+# it's the best you can do to put the global data in there.
+# In this case, I'll use this to store the values modifyable by the dashboard.
+class Data:
+    def __init__(self) -> None:
+        self.value = 0
+
+
+class Bot(hikari.GatewayBot):
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(**kwargs)
+        self.data = Data()
+
+
+# Startup the dashboard.
+# To see the dashboard, go to http://localhost:6361/ui/
+rillrate.install()
+bot = Bot(token=os.environ["DISCORD_TOKEN"], intents=hikari.Intents.ALL)
```

Comment:
The intents are not really needed for the example

Reply:
fixed on #eaa025dd